### ISSUE TITLE
Error: Memcache returned an error: %s

### ISSUE THREAD

**Issue by lushijie** [victorquinn/memcache-plus#39]
```
  let s1 = await memInst.set(key5, '10abc');
  await memInst.increase(key5);
```
when key's value is not integer, get the wrong error.
output : `Error: Memcache returned an error: %s`

**Comment by victorquinn** [comment]
Ugh that's not a very helpful error! Will investigate/fix

**Comment by toreau** [comment]
I've fixed this in https://github.com/victorquinn/memcache-plus/pull/54

**Comment by victorquinn** [comment]
This is published on npm as [0.2.20](https://www.npmjs.com/package/memcache-plus)
